thoth-client/assets/queries.graphql

```diff
@@ -0,0 +1,101 @@
+fragment Work on Work {
```

Comment:
This query is missing the field `reference` (as well as `updated_at`) from the set available in `WorkExtended`, and its fields are in a different order to that struct, making missing fields harder to spot. (I'm not sure what effect query field ordering has - the ordering of the fields in the CSV seems to be different yet again, ditto the ordering in `work/works_query.rs`.)

Reply:
I'm not entirely sure we want to recreate what's in `WorkExtended` right now - we can if we end up using the client in the app. Missing fields should not be an issue as we'll notice if/when we need them. Once we review the CSV output we'll see if folk want the `reference` included in there, otherwise there's not much point including it in the client. Ordering of the CSV will always be different to what we predefine in the structs (especially once we have more than one CSV output)